manticore/wasm/cli.py

```diff
@@ -0,0 +1,19 @@
+from .manticore import ManticoreWASM
+
+
+def wasm_main(args, _logger):
+    env = {key: val for key, val in [env[0].split("=") for env in args.env]}
```

Comment:
Are those environment variables or WASM env like imported functions? I guess this should be the latter - if so - why do we parse `key=value` string here? I mean, it could be passed as a dict instead?

Reply:
if this is one or the another: it is probably good to add a comment/docstring about it here or/and in `ManticoreWASM`.

Reply:
It seems this are envvars - hmm after rethinkinig it: maybe some wasm engines supports them (e.g. via `getenv`) so it might be reasonable to keep them; though it would probably be nice to explicitly add them to `ManticoreWASM.__init__` and document why/what is the reasoning beyond accepting them (imho it is not be that obvious straight away?).

Reply:
This was a mistake, at the time. I use `env` to pass in imported functions, but this was still using the code for passing in environment variables on the command line. At present, there's no way to specify imported functions via the CLI, since I'm not sure how that would work.